csharp/Worker/Microsoft.Spark.CSharp/Worker.cs

```diff
@@ -425,7 +425,9 @@ private object[] GetNext(int messageLength)
                     {
                         Debug.Assert(messageLength > 0);
                         byte[] buffer = SerDe.ReadBytes(inputStream, messageLength);
-                        result = unpickler.loads(buffer) as object[];
+                        var unpickledObjects = PythonSerDe.GetUnpickledObjects(buffer);
```

Comment:
Seems that a new Unpickler instance will get create by PythonSerDe.GetUnpickledObjects for every row received? 

If i understand this code path correctly, does the Unpicker instance need to live on outside this method for the whole lifetime of the Row being created, or is it only needed at this point of Row creation?

In other words, can Unpicker instances be cached & re-used for different Rows [maybe ThreadLocal?] to avoid creating so many new objects?